xwiki-platform-core/xwiki-platform-websocket/src/main/java/org/xwiki/websocket/internal/XWikiWebSocketResponseStub.java

```diff
@@ -185,15 +182,4 @@ public void addCookie(Cookie cookie)
 
         addHeader("Set-Cookie", header.toString());
     }
-
-    @Override
-    public void removeCookie(String cookieName, XWikiRequest request)
```

Comment:
Why was this method removed?

Reply:
Mainly because that method does not exist anymore in the interface (`XWikiWebSocketResponseStub` used to implement `com.xpn.xwiki.web.XWikiResponse` and is now implementing of `jakarta.servlet.http.HttpServletResponse`).